internal/v1/handler_compose_image.go

```diff
@@ -190,6 +204,7 @@ func buildRepositories(arch *distribution.Architecture, imageType ImageTypes) []
 	for _, r := range arch.Repositories {
 		// If no image type tags are defined for the repo, add the repo
 		if len(r.ImageTypeTags) == 0 || slices.Contains(r.ImageTypeTags, string(imageType)) {
+			fmt.Println("appending", *r.Baseurl, string(imageType))
```

Comment:
A leftover? Logging would be more appropriate.

Reply:
oh sure, ye